create-plugin-update/action.yml

```diff
@@ -0,0 +1,85 @@
+name: "Grafana Build Plugin"
```

Comment:
Is this name and description correct?

Reply:
Nope, copy n pasta mistake on my part. Should be fixed now.